package.json

```
@@ -2,18 +2,7 @@
   "name": "publish-browser-extension",
   "description": "Easily publish web extensions to their stores",
   "version": "3.0.2",
-  "engines": {
-    "node": "^18.0.0 || >=20.0.0"
-  },
-  "devEngines": {
-    "runtime": [
-      {
-        "name": "node",
-        "version": "20.17.0",
-        "onFail": "download"
-      }
-    ]
-  },
```

IDK if any bun tooling actually supports this, but pnpm, corepack, etc will this to warn you if you use the wrong one (npm is needed for publishing).

```suggestion
  "devEngines": {
    "runtime": [
      {
        "name": "bun",
        "version": "1.3.1",
        "onFail": "download"
      }
    ],
    "packageManager": [
       {
         "name": "bun",
         "version": "1.3.1",
         "onFail": "download"
       },
       {
         "name": "npm",
         "version": ">= 11.5.1"
       }
    ]
  },
```